bbb-learning-dashboard/public/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="/var/www/bigbluebutton-default/favicon.ico" />
```

Comment:
Hmmm, does this `%PUBLIC_URL%` need to be replaced when installing the package, or is it some magic keyword? cc @gustavotrott 
I have never seen an absolute path `/var/www/...` be used on served html (it would work if you open the html in your local browser of course).
Or maybe we construct the view on the server side and serve it? In which case the backend might actually be able to see the absolute path...
Could you please comment on this?

Reply:
Yes, this way will not work properly.
@JoVictorNunes, I guess you can just replace `%PUBLIC_URL%/favicon.ico` by `/favicon.ico`.
and it should work as expected!